odoo/fields.py

```diff
@@ -3034,7 +3034,7 @@ def _description_domain(self, env):
                 cids = 'company_ids'
                 field_to_check = 'company_ids'
             else:
-                _logger.warning(_(
+                _logger.warning(self.env._(
```

Comment:
`self` is a field...
```suggestion
                _logger.warning(env._(
```